src/dcommands/ccommand.js

```diff
@@ -655,7 +655,7 @@ class CcommandCommand extends BaseCommand {
                         });
                         if (command.length > 0) {
                             bu.send(msg, `Subcommand: **${command[0].name}**
-Aliases: **${command[0].aliases.join('**, **')}**
+Aliases: **${command[0].aliases ? command.aliases.join("**, **") : "none"}**
```

Comment:
`.aliases` is not a property of `command`. change this to `(command[0].aliases || []).join('**, **') || 'none'`

Reply:
I've forgotten an [0] in there!

Although, I'm not sure why you want 
`Aliases: **${(command[0].aliases || []).join("**, **") || 'none'}` 

instead of 

`Aliases: **${command[0].aliases ? command[0].aliases.join("**, **") : "none"}**`